backend/package.json

```diff
@@ -16,6 +16,7 @@
     "express": "~4.16.0",
     "firebase-admin": "^6.0.0",
     "morgan": "~1.9.0",
+    "node-fetch": "^2.2.0",
```

Comment:
This is a dev dependency, not a application dependency. We only use it for testing.